app/views/applications/show.html.erb

```diff
@@ -100,12 +100,23 @@
         class: "btn btn-delete",  method: :delete,
         data: { confirm: "Are you sure you want to delete this application?" } %>
   </div>
-<% elsif current_user.admin? || @application.submitted == false %>
+<% elsif @application.submitted == false %>
   <div>
     <%= link_to 'Delete', event_application_path(@event.id, @application.id),
         class: "btn btn-delete",  method: :delete,
         data: { confirm: "Are you sure you want to delete this application?" } %>
   </div>
+<% elsif current_user.admin? %>
+  <div>
+    <%= link_to 'Delete', event_application_path(@event.id, @application.id),
+        class: "btn btn-delete",  method: :delete,
+        data: { confirm: "Are you sure you want to delete this application?" } %>
+    <% if @event.open? && @application.unapproved && @application.submitted == false %>
```

Comment:
This condition will always be false because we are checking for  elsif @application.submitted == false above first.
Also - admins should not be able to see drafts. 

-> Needs to be  @application.submitted == true

Reply:
How about also 'revert' and 'reject' options for admins here?